tools/cli/factory.go

```diff
@@ -21,112 +21,87 @@
 package cli
 
 import (
-	"errors"
-
+	serverAdmin "github.com/uber/cadence/.gen/go/admin/adminserviceclient"
```

Comment:
Any reason to change it from "go.uber.org/cadence/.gen/go/admin/adminserviceclient" to "github.com/uber/cadence/.gen/go/admin/adminserviceclient"? Looks like the cadence-tools will break and I don't see an easy way to fix.